discovery-infra/download_logs.py

```diff
@@ -117,6 +118,18 @@ def is_update_needed(output_folder: str, update_on_events_update: bool, client:
     return need_update
 
 
+def download_and_save_files(files_list: list, client: InventoryClient, cluster: dict,
+                            output_folder: str, raise_exception:bool = True):
+    try:
+        for cluster_file in files_list:
+            with suppressAndLog(assisted_service_client.rest.ApiException):
```

Comment:
it the exception is suppressed then it would never be raised and be caught

Reply:
I see.. So if it is suppressed, how does it happen in the first place?

Reply:
That's why the function is named as "supress and log" :) you see the log output. the flow of the calling function isn't affected.

Reply:
I just saw it .. :) 
Maybe suppress and log is not the best approach here. I don't want to see the whole traceback in log if it's not an actual error

Reply:
@YuviGold What do you think change `suppressAndLog` to print a warning instead of full tarceback?

Reply:
I think that most of the time when there's an exception thrown I'd like to know where it comes from. So the traceback is necessary, otherwise, I might think it was something critical.
Suggestion - What do you thing about `suppressAndLog` writing `Suppressed exception from {func}: {message}`?

Reply:
@YuviGold  I don't really mind the syntax.  If we decide to suppress the exception, I think we should write it down so anyone who reads it will know that it was suppressed  :)

Reply:
So let's emphasize it's a suppressed exception. :)